provider/docker.go

```diff
@@ -688,6 +688,9 @@ func listTasks(ctx context.Context, dockerClient client.APIClient, serviceID str
 	var dockerDataList []dockerData
 
 	for _, task := range taskList {
+		if task.Status.State != swarm.TaskStateRunning {
+			continue
+		}
```

Comment:
One quick question why is this needed if we already [filtered](https://github.com/containous/traefik/pull/1304/files#diff-0e8a9f9593585fb3fa7ef0396fd5bbecR682) by `running` state before ?